lib/executor/constant_arrival_rate.go

```diff
@@ -264,7 +264,7 @@ func (car ConstantArrivalRate) Run(parentCtx context.Context, out chan<- metrics
 	}()
 
 	returnVU := func(u lib.InitializedVU) {
-		car.executionState.ReturnVU(u, true)
+		car.executionState.ReturnVU(u, false)
 		activeVUsWg.Done()
```

Comment:
This could generate confusion in the future, we set ReturnVU with _false_ but are setting **activeVUs**Wg as Done. Should we add a small comment to remind us why are we doing it? e.g. `We set activeVUs as false because the counter is already updated by the pool`.

Reply:
Good point. See 8d43a834.